AnnService/inc/Core/SPANN/ExtraFullGraphSearcher.h

```diff
@@ -192,6 +211,7 @@ namespace SPTAG
                         fileid = curPostingID / m_listPerFile;
                         listInfo = &(m_listInfos[fileid][curPostingID % m_listPerFile]);
                     }
+                    listInfo->postingID = curPostingID;
```

Comment:
this information can be assigned in the LoadHeadInfo

Reply:
Done